app/src/main/res/layout/fragment_home.xml

```diff
@@ -2,123 +2,155 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.constraintlayout.motion.widget.MotionLayout
+<androidx.coordinatorlayout.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/homeLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="?homeBackground"
-    app:layoutDescription="@xml/home_scene"
-    tools:context=".home.HomeFragment">
-
-    <ImageButton
-        android:id="@+id/privateBrowsingButton"
-        android:layout_width="@dimen/glyph_button_height"
-        android:layout_height="@dimen/glyph_button_height"
-        android:layout_marginEnd="12dp"
-        android:background="?android:attr/selectableItemBackgroundBorderless"
-        android:contentDescription="@string/content_description_private_browsing_button"
-        app:srcCompat="@drawable/private_browsing_button"
-        app:layout_constraintBottom_toTopOf="@+id/wordmark"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <ImageView
-        android:id="@+id/wordmark"
-        android:layout_width="wrap_content"
-        android:layout_height="80dp"
-        android:layout_marginStart="16dp"
-        android:layout_marginTop="60dp"
+    android:background="?homeBackground">
+
+    <com.google.android.material.appbar.AppBarLayout
```

Comment:
@mcomella  Honnestly feel the same.  Imo, one recyclerview would work fine with a different view holder that can handle its own scrolling behavior. As for a `ScrollView`, I do know they don't interact well with `recyclerview` performance wise. We'd have to go towards a `scrollview` but at that point, why not just create a viewholder specifically for the mozilla logo that can give a collapse feeling?

The `AppBarLayout` is essentially use to give the same feel as before ( since the `MotionLayout` `home_scene.xml` had this effect.